frontend/src/components/dashboard/aliases/BlockLevelSlider.tsx

```diff
@@ -103,14 +103,37 @@ export const BlockLevelSlider = (props: Props) => {
         <label {...labelProps} className={styles["slider-label"]}>
           {sliderSettings.label}
         </label>
-        <div {...trackProps} ref={trackRef} className={styles.track}>
-          <div className={styles["track-line"]} />
-          <div
-            className={getTrackStopClassNames(props.alias, sliderState, "none")}
-          >
-            <img src={UmbrellaClosedMobile.src} alt="" />
-            <p aria-hidden="true">{getLabelForBlockLevel("none", l10n)}</p>
+        <div className={styles["track-wrapper"]}>
```

Comment:
Essentially, the only change here is the new element `.track-wrapper`, wrwapping `.track`, and that `<PromotionalTrackStop>` was moved to below `.track` (but inside `.track-wrapper`).